skins/niculskin/index.html.tmpl

```diff
@@ -78,8 +78,22 @@
                         <tr><td>$niculskinLabels.conditions.wind</td><td>$current.windSpeed $current.windDir.ordinal_compass ($current.windDir)</td></tr>
                         <tr><td>$niculskinLabels.conditions.rain_rate</td><td>$current.rainRate</td></tr>
                         <tr><td>$niculskinLabels.conditions.in_temp_humidity</td><td>$current.inTemp | $current.inHumidity</td></tr>
+                        #if $day.extraTemp1.has_data
                         <tr><td>$niculskinLabels.conditions.extra1_temp_humidity</td><td>$current.extraTemp1 | $current.extraHumid1</td></tr>
+                        #end if
+                        #if $day.extraTemp1.has_data
                         <tr><td>$niculskinLabels.conditions.extra2_temp_humidity</td><td>$current.extraTemp2 | $current.extraHumid2</td></tr>
+                        #end if
+                        #if $day.UV.has_data
+                        <tr><td>UV</td><td>$current.UV</td></tr>
```

Comment:
Please use a reference to the label. The english value is defined in the skin.conf starting at [line 266](https://github.com/ngulden/niculskin/blob/d534d27e9616359fa150a6fc20e1f39ca07fe9b2/skins/niculskin/skin.conf#L266). For example add a new line after line 286 with uv, et and radiation.

This way the template remains translatable and other languages can easily be added to the appropriate language skin subdirectory.